source/_components/notify.facebook.markdown

```diff
@@ -117,3 +117,87 @@ script:
                 title: Blue
                 payload: DEVELOPER_DEFINED_PAYLOAD_FOR_PICKING_BLUE
 ```
+
+You can now also use Facebook public beta broadcast API to push messages to ALL users who interacted with your chatbot on your page, without having to collect their number. This will scale to thousands of users. Facebook requires that this only be used for non-commercial purposes and they validate every message you send. Also note, your Facebook bot needs to be authorized for "page_subscritions" if you want to make it to all, but can be used right away to a selected group of testers of your choice. 
+
+To enable broadcast just use the keyword "BROADCAST" as your target. Only put ONE target BROADCAST as below:
+
+```yaml
+- alias: Facebook Broadcast
+  trigger:
+    platform: sun
+    event: sunset
+  action:
+    service: notify.<FACEBOOK_NOTIFY_COMPONENT_NAME_YOU_CREATED>
+    data:
+      message: <TEXT_YOU_ARE_SENDING>
+      target:
+        - BROADCAST
+```
+
+Here is an advanced sample for broadcasting wind alerts from a wunderground weather station:
+
+configuration.yaml:
+
+```yaml
+notify:
+  - platform: facebook
+    name: facebook_wind_alerts
+    page_access_token: !secret facebook_page_access_token  
+
+ platform: wunderground
+ api_key: !secret wunder_api_key
+ pws_id: !secret  wunder_station_id
+ monitored_conditions:
+    - wind_kph
+    - wind_dir
+    - wind_gust_kph
+    - pressure_trend
+    - wind_string
+    - weather_1h
+ 
+ 
+ platform: template
+ sensors:
+    windy:
+      friendly_name: 'Wind'
+      icon_template: mdi:weather-windy
+      value_template: >-
+        {% if (states.sensor.pws_wind_kph.state | float + states.sensor.pws_wind_gust_kph.state | float)/2.0 > 18.0 -%}
```

Comment:
All templates must be escaped otherwise they are rendered empty.

Reply:
@fabaff I am very confused about this request. the documentation is very clear that ONLY single line templates must be escaped. these are NOT single line. And also, they work quite well on my home installation. So they do not render empty, I guarantee you. Please take a look at the documentation here: https://home-assistant.io/docs/automation/templating/

If you insist, I will put double quotes there, but it would be wrong in my view.

Reply:
@fabaff I guess this one is single line. I will change it.